package.json

```diff
@@ -8,13 +8,14 @@
         "lint:fix": "prettier-eslint **/**/*.{js,json} --write",
         "prettier": "prettier **/**/*.{js,json} --write",
         "test": "jest --forceExit",
+        "build": "babel server --out-dir build",
         "dev": "nodemon --exec babel-node server/index",
         "migrate": "knex migrate:latest",
         "seed": "babel-node server/seeds/index.js",
         "make:migration": "knex migrate:make",
         "migrate:rollback": "knex migrate:rollback",
-        "start": "node server/index",
-        "heroku-postbuild": "npm run migrate",
+        "start": "node build/index",
```

Comment:
I'm thinking about this script. I think someone else already did this. not sure. I'll look into all the other open PRs

Reply:
what should i do about it.....remove it?

Reply:
left it